### PR TITLE
[12.0] [REF] Queue Job: allow configuration from server_environment

### LINKED ISSUE #62: Making queue_job aware of server_environment
Hi,

It would be nice to make the jobrunner configuration compatible with server_environment.

Our motivation is to push the config with code and avoid risks of deployment errors when deployment is not under our direct control.

I was wondering if something simple, albeit a bit hackish, would make sense [here](https://github.com/OCA/queue/blob/5c098fc3340f7b837cbb5da18ea4d0fa9aadc37c/queue_job/jobrunner/runner.py#L131):

```
try:
    from openerp.addons.server_environment import serv_config as config
except ImportError:
    from odoo.tools import config
```

@guewen @lmignon

### PR DESCRIPTION
**Pull request by ThomasBinsfeld**
Fixes #62

### CODE REVIEW

File: queue_job/jobrunner/runner.py [hunk 1]
```diff
@@ -146,7 +146,15 @@
...
-from odoo.tools import config
+try:
+    from odoo.addons.server_environment import serv_config as config
+    if config.has_section('queue_job'):
+        queue_job_config = config['queue_job']
+    else:
+        queue_job_config = {}
+except ImportError:
+    from odoo.tools import config
+    queue_job_config = config.misc.get('queue_job', {})
```
Comment (guewen) [inline queue_job/jobrunner/runner.py]: This is the same code than the one in `__init__.py`, could you import only `queue_job_config` from a common place?

**Review by guewen — CHANGES_REQUESTED** [review]
Besides the replication of the import logic, it looks good, thanks!

**Comment by lmignon** [comment]
@ThomasBinsfeld Can you take care of the comments expressed by guewen

File: queue_job/jobrunner/runner.py (hunk 1 above)
Comment (ThomasBinsfeld) [reply queue_job/jobrunner/runner.py]: Done

**Comment by ThomasBinsfeld** [comment]
ping @lmignon

**Review by guewen — APPROVED** [review]

**Review by lmignon — APPROVED** [review]

**Review by sbidoul — APPROVED** [review]

**Pull request merged**